TestProjects/Playground/Playground.Core/ViewModels/DictionaryBindingViewModel.cs

```diff
@@ -0,0 +1,48 @@
+using MvvmCross.Core.Navigation;
+using MvvmCross.Core.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Playground.Core.ViewModels
+{
+    public class DictionaryBindingViewModel : BaseViewModel
+    {
+        int _value = 0;
+        public int Value
+        {
+            get => _value;
+            set => SetProperty(ref _value, value);
+        }
+
+        IMvxAsyncCommand _closeCommand;
+        public IMvxAsyncCommand CloseCommand =>
+            _closeCommand ?? (_closeCommand = new MvxAsyncCommand(async () => await _navigationService.Close(this)));
+
+
+        IMvxCommand _incrimentCommand;
+        private IMvxNavigationService _navigationService;
+
+        public IMvxCommand IncrimentCommand =>
```

Comment:
Minor typo: `IncrementCommand` (similar in the private member and in the method used to create the command)